site/how-it-works.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Understanding Weave Scope
+menu_order: 70
+---
+
+The following topics are discussed: 
+
+* [Scope Reporting in Service Mode](#scope-reporting-in-service-mode)
+* [Scope Reporting in Standalone Mode](#scope-reporting-in-stand-alone-mode)
+* [Installing Weave Scope in Standalone Mode](#installing-weave-scope-standalone)
+* [Managing Clusters with Scope and weaveDNS](#managing-clusters-with-scope-and-weavedns)
+
+Weave Scope consists of two components: the app and the probe. The components are deployed as a single Docker container using the scope script. The probe is responsible for gathering information about the host on which it is running. This information is sent to the app in the form of a report. The app processes reports from the probe into usable topologies, serving the UI, as well as pushing these topologies to the UI.
+
+
+## <a name="scope-reporting-in-service-mode"></a>Scope Reporting in Service Mode
+
+Scope can also be used to feed reports to the Scope Service. The Scope Service allows you centrally manage and share access to your Scope UI. In this configuration, the probe is run locally and the apps are hosted for you.
+
+!['Service Mode Reporting'](service-mode.png)
+
+##<a name="scope-reporting-in-stand-alone-mode"></a>Scope Reporting in Standalone Mode
+
+When running Scope in a cluster, each probe sends its reports to a dedicated app. The app merges the reports from its probe into a comprehensive report that is sent to the browser.  To visualize your entire infrastructure and apps running on that infrastructure, Scope must be launched on to every machine you are using.
+
+!['Stand-alone Reporting'](stand-alone.png)
+
+##<a name="installing-weave-scope-standalone"></a>Installing Weave Scope Standalone
```

Comment:
I would move this to the introduction as this is what most people will be looking for.

I would add a line at the bottom of this section, in the introduction, saying "for instructions on how to install scope on other platforms, see..."